src/Model/ApiResource.php

```diff
@@ -19,11 +20,14 @@
  */
 abstract class ApiResource
 {
+
+    use TimestampableEntity;
```

Comment:
We can add the timestample trait here, but we also have to add "gedmo/doctrine-extensions" to the composer.json-